.ci-operator.yaml

```diff
@@ -1,4 +1,4 @@
 build_root_image:
   name: release
   namespace: openshift
-  tag: rhel-8-release-golang-1.17-openshift-4.10
+  tag: rhel-8-release-golang-1.20-openshift-4.14
```

Comment:
At some point I wonder if we should squash these updates into the set up commit that we are carrying